tests/playwright/pages/elementor-panel-tabs/content.js

```diff
@@ -8,9 +8,13 @@ export default class Content {
 		this.editorPage = new EditorPage( this.page, testInfo );
 	}
 
+	async setLinkTo( option ) {
```

Comment:
Won't this be confusing along with 'setLink'?

Reply:
fixed